index.html

```diff
@@ -457,5 +457,69 @@ <h3>
 
 </div>
 
+
+<div id="heading-grid">
+    <h4>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h4>
+</div>
+
+<form id="form-grid" action="https://formspree.io/f/xvodvrjv"
+method="POST">
+    <input type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+    <input type="email" placeholder="Email address" name="email" required>
+    <textarea placeholder="Enter text here" name="comment-section" maxlength="500" required>Enter text here</textarea>
```

Comment:
Please, remove the default text "Enter text here" and add it as a placeholder value for your `textarea` element, for instance:
```
placeholder="Enter text here"
```

- [ ] Remove default text in your textarea, and add this default text as placeholder value.